db/fixtures/miq_searches.yml

```diff
@@ -1,103 +1,103 @@
 ---
-- attributes: 
```

Comment:
I think it's fine...if it's not a lot of effort to put back, then yeah, maybe undo it and have a follow up PR, but it's up to you.